shell/pages/c/_cluster/explorer/index.vue

```diff
@@ -149,6 +151,10 @@ export default {
     ...monitoringStatus(),
 
     displayPspDeprecationBanner() {
+      if (!this.currentCluster) {
```

Comment:
@Shavindra The changes in this file don't seem to relate to Fleet - what's the bug/case these changes address? Are they needed? We should file an issue if there is a bug that we're fixing here.